UIP.py

```diff
@@ -1,12 +1,11 @@
 from scheduler import scheduler
-
+    

```

Comment:
@vishvanath45 I think it's better if you use a text editor that shows up git diff in its UI. So that you avoid unnecessary changes in files, and do leave blank lines as is. My preferred text editor is atom, developed by github.